python_multipart/multipart.py

```diff
@@ -1666,26 +1688,29 @@ def on_headers_finished() -> None:
                 is_file = False
 
                 # Parse the content-disposition header.
-                # TODO: handle mixed case
-                content_disp = headers.get(b"Content-Disposition")
+                content_disp = headers.get("content-disposition")
                 disp, options = parse_options_header(content_disp)
 
                 # Get the field and filename.
-                field_name = options.get(b"name")
+                field_name = options.get(b"name", b"")
```

Comment:
Is this needed? Isn't the `field_name` supposed to always be present?

Reply:
I added this for typing. Without it, we'll need to raise or assert it's not None. Which makes sense actually.
Will probably need a test to cover it. too.

Reply:
Removed the default value and added a check for `None`.